Mathlib/GroupTheory/Perm/Cycle/Concrete.lean

```diff
@@ -527,7 +527,7 @@ def isoCycle' : { f : Perm α // IsCycle f } ≃ { s : Cycle α // s.Nodup ∧ s
 
 -- mutes `'decide' tactic does nothing [linter.unusedTactic]`
 set_option linter.unusedTactic false in
-notation3 (prettyPrint := false) "c["(l", "* => foldr (h t => List.cons h t) List.nil)"]" =>
+local notation3 (prettyPrint := false) "c["(l", "* => foldr (h t => List.cons h t) List.nil)"]" =>
```

Comment:
I think this would make more sense as scoped notation (`open scoped Equiv.Perm`), rather than local notation.